components/dashboard/src/teams/TeamNetworking.tsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { isGitpodIo } from "../utils";
+import React from "react";
+import { Heading2, Heading3, Subheading } from "../components/typography/headings";
+import { OrgSettingsPage } from "./OrgSettingsPage";
+import { ConfigurationSettingsField } from "../repositories/detail/ConfigurationSettingsField";
+import { useDocumentTitle } from "../hooks/use-document-title";
+import { LinkButton } from "@podkit/buttons/LinkButton";
+import { CheckCircle2Icon } from "lucide-react";
+import { Redirect } from "react-router";
+import PillLabel from "../components/PillLabel";
+
+export default function TeamPoliciesPage() {
+    useDocumentTitle("Organization Settings - Networking");
+
+    if (!isGitpodIo) {
+        return <Redirect to="/settings" />;
+    }
+
+    return (
+        <>
+            <OrgSettingsPage>
+                <div className="space-y-8">
+                    <div>
+                        <Heading2 className="flex items-center gap-4">
+                            Networking
+                            <PillLabel type="warn" className="!px-3 !py-0.5 !text-xs border">
+                                <span className="text-sm font-semibold text-pk-content-secondary">Enterprise</span>
+                            </PillLabel>
+                        </Heading2>
+                        <Subheading className="mt-1">Manage your organization's networking settings.</Subheading>
+                    </div>
+
+                    <SelfHostedCalloutCard />
+                    <DeployedRegionCard />
+                    <VPNCard />
+                </div>
+            </OrgSettingsPage>
+        </>
+    );
+}
+
+const SelfHostedCalloutCard = () => {
+    return (
+        <ConfigurationSettingsField className="bg-pk-surface-secondary">
+            <Heading3>Self-host in your cloud account</Heading3>
+            <Subheading className="mt-1">
+                Deploy the Gitpod infrastructure into your own cloud account and connect to your private network
+            </Subheading>
+
+            <div className="mt-1 flex flex-col space-y-2">
```

Comment:
I know the designs have different spacing, but it wasn't intentional 🙏 

```suggestion
            <div className="mt-8 flex flex-col space-y-2">
```